File: leetcode/7. Reverse Integer/7_reverse_integer.py
class Solution:
    def reverse(self, x: int) -> int:
        sign = 1 if x > 0 else -1
        value = sign * int(str(x*sign)[::-1])
        
        if value not in range(-2**31, 2**31):
            value = 0
            
        return value

File: leetcode/7. Reverse Integer/7_reverse_integer.cs
public class Solution {
    public int Reverse(int x) {

        long answer = 0;
        while(x != 0)
        {
            answer = answer * 10 + x % 10;
            x /= 10;
        }
        return Math.Abs(answer) > Int32.MaxValue ? 0 : (int)answer;
    }
}
